dts/bindings/clock/st,stm32-hse-clock.yaml

```diff
@@ -13,3 +13,14 @@ properties:
     description: |
       HSE crystal oscillator bypass
       Set to the property to by-pass the oscillator with an external clock.
+
+  css-enabled:
+    type: boolean
+    description: |
+      HSE clock security system enabled
+      Set the property to enable the clock security system for the HSE clock.
+
+      If a failure is detected on the HSE clock, the HSE oscillator is automatically disabled,
```

Comment:
Open question: what is the behavior if HSE clock is disabled by Power Mgmt ?

Reply:
That's a good question. The G4 reference manual just says that CSS "is
enabled after the HSE oscillator startup delay, and disabled when this oscillator is stopped". The H7 reference manual says "The CSS on HSE is disabled when the HSE is disabled. As a result, this function does not
work when the system is in Stop mode."

In neither case is it clear what happens when HSE is enabled again. Is CSS also re-enabled or does it have to be re-enabled in software? Do you have any contacts at ST who could clarify?

On the other hand, I think it doesn't matter for this. The `set_up_fixed_clock_sources` where I added the code to enable CSS is called from `stm32_clock_control_init` and according to the comment at the top of that function (in clock_stm32_ll_common.c) it's "called on the startup and also to restore the config when exiting for low power mode".

The lazy solution would be to just add a comment saying that CSS behaviour around power management has not been tested and users need to verify it themselves on their own hardware.

Reply:
@erwango What do you think would be the best way to resolve this?

Reply:
> The lazy solution would be to just add a comment saying that CSS behaviour around power management has not been tested and users need to verify it themselves on their own hardware.

Ok, fine for me if you fine a nice wording.

Reply:
> > The lazy solution would be to just add a comment saying that CSS behaviour around power management has not been tested and users need to verify it themselves on their own hardware.
> 
> Ok, fine for me if you fine a nice wording.

I've just pushed an update to the binding description. Do you think this is OK?